state/azure/cosmosdb/cosmosdb.go

```diff
@@ -413,9 +407,7 @@ func (c *StateStore) Multi(request *state.TransactionalStateRequest) (err error)
 
 	c.logger.Debugf("#operations=%d,partitionkey=%s", numOperations, partitionKey)
 
-	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
```

Comment:
Please don't remove the background. Still use a context from `ctx` with the timeout

Reply:
Ditto.

Reply:
Fixed.